content/vi/toc.md

```diff
@@ -0,0 +1,38 @@
+12 hệ số
+========
+
+## [I. Mã gốc](./codebase)
+### Một mã gốc được theo dõi với hệ thống quản lý phiên bản, và nhiều lần triển khai
+
+## [II. Các phụ thuộc](./dependencies)
+### Khai báo rõ ràng và phân tách các phụ thuộc
+
+## [III. Cấu hình](./config)
+### Lưu trữ cấu hình trong môi trường
+
+## [IV. Dịch vụ hỗ trợ](./backing-services)
+### Dịch vụ hỗ trợ như là tài nguyên bổ xung
+
+## [V. Xây dựng, phát hành, vận hành](./build-release-run)
+### Tách biệt hoàn toàn giữa bước xây dựng và vận hành
+
+## [VI. Tiến trình](./processes)
+### Vận hành ứng dụng như là một hoặc nhiều tiến trình phi trạng thái 
+
+## [VII. Mở cổng mạng](./port-binding)
```

Comment:
I think this translated term is a bit ambiguous. Your translation literally means "Opening Ports".  What do you think about "Gán cổng"?

Reply:
I translated word by word. "Gán cổng" is better.